nrfx/samples/src/nrfx_egu/nrfx_build_boilerplate.cmake

```diff
@@ -0,0 +1,13 @@
+set(REQUIRES
+    NRF52810_XXAA
+    NRF52811_XXAA
+    NRF52820_XXAA
+    NRF52832_XXAA
+    NRF52833_XXAA
+    NRF52840_XXAA
+    NRF9160_XXAA
+    NRF5340_XXAA_APPLICATION
+    NRF5340_XXAA_NETWORK
+)
+
+include(../../common/nrfx_build_boilerplate_common.cmake)
```

Comment:
Are these boilerplate files needed in this repo? `nrfx_build_boilerplate_common.cmake` references a path that does not exist in the Zephyr environment.